src/commands/sbt_cached.yml

```
@@ -50,6 +56,8 @@ steps:
               set +eo pipefail
               find $HOME/.sbt -name "*.lock" | xargs rm | true
               find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm | true
+              find -name target -exec tar -zcf ~/targets.tar.gz -H posix {} + | true
```

just to be safe wouldn't it be better to find only directories?

```suggestion
              find -name target -type d -exec tar -zcf ~/targets.tar.gz -H posix {} + | true
```